mypy/stubgen.py

```diff
@@ -1507,9 +1507,7 @@ def is_blacklisted_path(path: str) -> bool:
 
 
 def normalize_path_separators(path: str) -> str:
-    if sys.platform == "win32":
```

Comment:
If https://github.com/python/mypy/pull/18539 or a similar fix is merged, then this change should no longer be necessary I believe.